src/components/minor-button/minor-button.stories.mdx

```diff
@@ -5,10 +5,12 @@ import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
 
 <Meta title="Minor Button" parameters={{ info: { disable: true } }} />
 
-# Minor Button
+# Button
 
-A Button triggers a single action or event.
-Use it to submit a form (Save), to advance to the next step in a process (Next), or to create a new item (New).
+- A minor button is a user interface button rather than a primary action button.
+- It is less prominent than a major button.
+- Use minor buttons when displaying many buttons on a page.
+- Group minor buttons together to form a button bar.
```

Comment:
Thanks for adding this @DobroTora 👍🏼 

I feel we can tweak the wording a little bit so its clearer, but perhaps it would be better if this is something @harpalsingh or Design System would be able to assist you with 😄

Reply:
@Parsium I requested our content writer opinion and check from @emmacassidy

Reply:
@Parsium just to let you know, we've been deliberately deleting out all design guidance from every carbon page on a separate PR. Part 2 of this work is to cross-link to the design system website where we can maintain designer notes like this in one place. 

With that in mind, i don't think we should be adding this in as it undos the work we are doing elsewhere.

Reply:
So the content writer - @emmacassidy  suggested to keep 2 first bullet points as we are migrating Designer's Notes into ZH and the other 2 are part of it.

Reply:
Ah awesome - thanks for clarifying this @ljemmo @DobroTora 👍🏼